svc-aggregation/README.md

```diff
@@ -210,8 +331,9 @@ curl -i POST \
 |PluginID|String \(required\)<br> |The id of the plugin you want to add. Example: GRF \(Generic Redfish Plugin\), ILO<br> |
```

Comment:
remove PluginID, PreferredAuthType and PluginType from the Request parameters

Reply:
Removed